### ISSUE TITLE
Check extra whitespace success, but check extra tab character in the begin of line does not work

### ISSUE THREAD

**Issue by ElenaSpb** [pinterest/ktlint#425]
gradle task ktlint failed if there are extra **whitespaces** with 
_error line="9" column="1" severity="error" message="Unexpected indentation (11) (it should be 8) (cannot be auto-corrected)" source="indent"_ 
for example.

but if there are extra **tab character** in the begining of line - the task finish success, which is not expected.

**Comment by shashachu** [comment]
@ElenaSpb can you give a small code snippet demonstrating the issue? Thanks!

**Comment by realdadfish** [comment]
I'm not sure if this is related, but this code snippet does not provoke an error in KtLint, which it should:

```kotlin
class Foo {
    private fun doBar(): Int =
        if (somethingIsTrue) {
            doThis()
    } else {
        doThat()
    }
    ...
}
```

Notice how the indentation of the `else` block is not following the one of the `if` block. This is with ktlint 0.34.2 btw. and was also the case with 0.31 and earlier versions before.

**Comment by alejandrorosas** [comment]
I think he means this case:
```
class Foo {
    private fun doBar() {
            doThis()
    }

    private fun doFoo() {
        doThis()
    }
    ...
}```

So this case it succeeds when linting although there's an extra tab on the `doBar()` method.

**Comment by ElenaSpb** [comment]
thanks a lot, just I already config ktlint properly and it works fine.

**Comment by ElenaSpb** [comment]
only if package in class is not the same as location - it does not show such errors..

**Comment by ggajews** [comment]
@ElenaSpb I run into the same issue. How did you change the configuration to fix this?

**Comment by Tapchicoma** [comment]
@ggajews is it happening for you as well?

**Comment by ElenaSpb** [comment]
execuse me - I thought I leave comment in another issue.
this 2 bugs which I describe are still reproduced
may be there were made some changes 
but in 
ktlint "com.github.shyiko:ktlint:0.31.0"
it is reproduces

**Comment by ggajews** [comment]
@Tapchicoma yes, we are using spaces for formatting, and if I add a random tab in the line, ktlint does not report it.

**Comment by paul-dingemans** [comment]
Problem is that lines that are properly indented at the correct level but are using the wrong indentation style (tabs instead of space or vice versa) are not autocorrected. In the example of this issue, only the the improperly indented lines were fixed. PR is on its way.
